[feat] implemented custom OSD position

yuxiaolejs opened the pull request:
implemented custom OSD position by adding extra field in line 1267

yuxiaolejs commented:
@agsh

agsh commented:
@yuxiaolejs Hi! Nice PR! Please just add also jsdoc to describe all the possible parameters of `position` argument: https://github.com/yuxiaolejs/onvif/blob/2a803e33ab22f2a8cf04ab683db3863d27f1a3d6/lib/media.js#L1251

agsh commented on lib/media.js:
```diff
@@ -1263,7 +1263,8 @@ module.exports = function(Cam) {
             <sch:Position>
-               <sch:Type>${options.position || 'LowerLeft'}</sch:Type>
+               <sch:Type>${typeof (options.position) == "object" ? "Custom" : options.position}</sch:Type>
+               ${typeof (options.position) == "object" ? ("<sch:Pos x=\"" + options.position.x + "\" y=\"" + options.position.y + "\"/>") : ""}
```
And here you can use strict equal: `typeof (options.position) === 'object'`

yuxiaolejs commented:
@agsh  Hi! I updated the docstring, and also changed it to strict equals. Have the code tested on my Hikvision DS-2CD3325-I IPC, everything worked as expected

agsh commented:
@yuxiaolejs Cool! Should I put the updated version to NPM?

agsh commented:
@yuxiaolejs Oh, and also fix lint errors, please: https://github.com/agsh/onvif/actions/runs/3729451314

yuxiaolejs commented:
> @yuxiaolejs Cool! Should I put the updated version to NPM?

I am glad to hear that! It's my pleasure to contribute to this project, and seeing it updated on NPM is great!

agsh commented:
@yuxiaolejs Great! Just make a new PR with all of the changes please (I've reverted the merge because the linting pipeline broke) :) And I'll do the rest.
